### ISSUE TITLE
"Remove entry" issues

### ISSUE THREAD

**Issue by hotice** [SergKolo/launcher-list-indicator#1]
When the "Remove entry" indicator menu item is used, a popup to name the profile is displayed.  If I name the profile with the exact same name as an existing profile (either the one I want to remove or another one), all profile entries disappear from the indicator. The ~/.launcher_profiles.json file continues to have the other profiles, they just don't show up in the indicator.

Also, if after clicking "Remove entry", I enter a profile name that's not already in use, nothing occurs (both the indicator and the configuration file remain unchanged).

**Comment by SergKolo** [comment]
@hotice Does it display any traceback issues when running from terminal ? I had the same issue in mouse-profile-indicator, which was fixed, and ported the fix into this indicator as well. The latest commit was `944f56d389aff8acc94629d81148ba42469dfe41` , run `git status` and see if you got the latest one.  With the latest commit I had changed the remove entry dialog to say "Which entry to remove?" instead of "Name the profile".
